torch/_higher_order_ops/invoke_subgraph.py

```diff
@@ -41,8 +41,8 @@ def __call__(
         subgraph: GraphModule,
         identifier: Optional[str],
         operands: Union[
-            List[Union[torch.Tensor, torch.SymInt]],
-            Tuple[Union[torch.Tensor, torch.SymInt]],
+            List[Union[torch.Tensor, int]],
```

Comment:
This is `__call__` - not while tracing. So, symints would not be present here, they would be converted to ints.

Reply:
Yeah, I just got that. Adding symints here as well.